src/status.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-mutable-exports */
+import moreIcon from './more.svg';
+// eslint-disable-next-line import/no-cycle
+import { todoList, todoTasks, elementGenerator } from './index';
+
+function savedList() {
+  localStorage.setItem('ToDo', JSON.stringify(todoTasks));
+}
+
+function statusUpdate(item, input, oneTodo, todoTasks) {
+  if (item.checked) {
+    input.classList.add('line-through');
+  } else {
+    input.classList.remove('line-through');
+  }
+
+  oneTodo.addEventListener('change', () => {
+    const todo = todoTasks[item.id];
+    todo.checked = !item.checked;
+    todo.completed = !item.completed;
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] I would suggest fixing this in code instead of `disabling linters`. All rules are applied for the purpose to write code in a best practice manner.

Reply:
Thank you, I have refactored my code. This is now fixed